apps/coordinator/docker-compose.yml

```diff
@@ -1,4 +1,16 @@
 services:
+  coordinator-database:
+    container_name: coordinator-database
+    image: redis:latest
+    restart: unless-stopped
+    env_file:
+      - ./.env
+    ports:
+      - "${COORDINATOR_REDIS_PORT:-6379}:${COORDINATOR_REDIS_PORT:-6379}"
+    command: redis-server --save 20 1 --loglevel warning
```

Comment:
What do 20 and 1 mean here?

Reply:
It means that redis will persist the nada if at least 1 write operation has ocurrend within 20 seconds